platform/client/src/pages/ViewActionPage.tsx

```diff
@@ -85,7 +85,18 @@ export const ViewActionPage: React.FC<RouteComponentProps<IUpdateMatchParams>> =
                 <AutoFields />
                 <ErrorsField />
               </AutoForm>
-              <Map lat={model.distributionCentre?.lat!} long={model.distributionCentre?.long!} name={model.distributionCentre?.name!}></Map>
+              <Map center={{
+                    lat: parseInt(model.distributionCentre?.lat!),
```

Comment:
If distribution centre is gone entire map should not be rendered. This will simplify logic

Reply:
Yeap. We should probably check if we have a distribution centre.